pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -472,6 +463,16 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return reconcile.Result{}, nil
 	}
 
+	if err := validateProfileMachineConfigPool(instance, profileMCP); err != nil {
+		conditions := r.getDegradedConditions(conditionBadMachineConfigLabels, err.Error())
+		if err := r.updateStatus(instance, conditions); err != nil {
+			klog.Errorf("failed to update performance profile %q status: %v", instance.Name, err)
+			return reconcile.Result{}, err
+		}
+
+		return reconcile.Result{}, nil
+	}
+
```

Comment:
from where this comes from?

Reply:
@ffromani   this is a good question ,   I saw it was removed (or moved) in [#854](https://github.com/openshift/cluster-node-tuning-operator/pull/854/files#diff-b26e8713b0dcfb154cd6a3430e574a73f0b601dc1a206b4fcd732e587d4fa38aL475)   to the[ v1 reconciler](https://github.com/openshift/cluster-node-tuning-operator/pull/854/files#diff-b26e8713b0dcfb154cd6a3430e574a73f0b601dc1a206b4fcd732e587d4fa38aR572) in that same PR. 
It looks like it was missing for the non v1 reconcile loop in #854 . Would that statement be correct ?

Reply:
Yep. Moved from line 599 in this patch.

Reply:
ok, LGTM then